packages/fx-core/src/component/driver/deploy/azure/impl/AzureZipDeployDriverImpl.ts

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureDeployDriverImpl } from "./azureDeployDriverImpl";
+import {
+  AxiosZipDeployResult,
+  AzureUploadConfig,
+  DeployStepArgs,
+} from "../../../interface/buildAndDeployArgs";
+import { AzureResourceInfo, DriverContext } from "../../../interface/commonArgs";
+import { TokenCredential } from "@azure/core-auth";
+import { IProgressHandler, LogProvider, UserInteraction } from "@microsoft/teamsfx-api";
+import { getLocalizedMessage, ProgressMessages } from "../../../../messages";
+import { DeployConstant } from "../../../../constant/deployConstant";
+import { createHash } from "crypto";
+import { default as axios } from "axios";
+import { DeployExternalApiCallError } from "../../../../error/deployError";
+import { HttpStatusCode } from "../../../../constant/commonConstant";
+
+export class AzureZipDeployDriverImpl extends AzureDeployDriverImpl {
```

Comment:
If it is only about Deploy interface, I would prefer to remove Driver concept in the naming to make it more clear. e.g. AzureDeployDriverImpl => AzureDeployImpl